wsit/ws-sx/wssx-impl/src/main/java/com/sun/xml/wss/impl/transform/AttachmentCompleteTransform.java

```diff
@@ -51,7 +51,7 @@ protected XMLSignatureInput enginePerformTransform(XMLSignatureInput input, Outp
             return new XMLSignatureInput(_canonicalize(input));
        } catch (Exception e) {
             // log
-            throw new TransformationException(e.getMessage(), e);
+            throw new TransformationException(e, e.getMessage());
```

Comment:
This is weird change. All standard exceptions have the message first ...

Reply:
tell that to the Santuario team... see https://santuario.apache.org/Java/api/org/apache/xml/security/transforms/TransformationException.html

Reply:
I see ... :-/